bandit/core/manager.py

```diff
@@ -249,6 +249,8 @@ def discover_files(self, targets, recursive=False, excluded_paths=""):
                     excluded_path_globs,
                     enforce_glob=False,
                 ):
+                    if fname != "-":
+                        fname = os.path.join(".", fname)
```

Comment:
So let's say someone does something like `bandit src/bandit/__init__.py`, we still need to prepend this?

Reply:
No, we don't need to prepend in that case, but it also does no harm.